tests/token/test_rest.py

```diff
@@ -3,23 +3,32 @@
 
 import pytest
 
-from spylib import Store
+from spylib import Token
 from spylib.exceptions import ShopifyCallInvalidError
-
-from ..shared import MockHTTPResponse
+from spylib.token import OfflineTokenResponse
+from tests.token.conftest import AppInformation
 
 
 @pytest.mark.asyncio
-async def test_store_rest_happypath(mocker):
-    store = Store(store_id='TEST', name='test-store', access_token='Te5tM3')
+async def test_store_rest_happypath(
+    mocker,
+    offline_token_data: OfflineTokenResponse,
+    app_information: AppInformation,
+    MockHTTPResponse,
+):
+    token = Token(
+        store_name=app_information.store_name,
+        access_token=offline_token_data.access_token,
+        scope=offline_token_data.scope.split(','),
+    )
```

Comment:
The token should be initialized like this only during the OAuth. Use `load` instead on an Offline or Online token (no need to do both). That's the point of defining the `test_database` in `conftest`, to test things the way they will be used by a user.
Same with the other Token initializations.